api/plugin-text.api.md

```diff
@@ -34,10 +32,11 @@ export interface BlockRendererProps {
 }
 
 // @public (undocumented)
-export function createTextPlugin({ placeholder, registry, theme }: {
+export function createTextPlugin({ placeholder, registry, theme, blockquote }: {
```

Comment:
Slightly breaking change. Text Plugin now accepts the type of blockquote plugin as part of its config. If it's not provided, we don't show the blockquote controls.

Reply:
nice